rebar.config

```diff
@@ -71,7 +73,7 @@
   {cpool, "0.1.0"},
   {observer_cli, "1.5.4"},
   {nkpacket, {git, "https://github.com/michalwski/nkpacket.git", {ref, "f7c5349"}}},
-  {nksip, {git, "https://github.com/NetComposer/nksip.git", {ref, "1a29ef3"}}},
+  {nksip, {git, "https://github.com/arcusfelis/nksip.git", {branch, "mu-fix-types-1a29ef3"}}},
```

Comment:
please submit PR with these fixes to the original repo.

Reply:
We use nksip from 5 years ago:

```
commit 1a29ef3e112ee0a8d8ac53bd6963ccc8f7879343
Author: Carlos Gonzalez <carlosj.gf@gmail.com>
Date:   Fri Nov 4 20:13:40 2016 +0100

    Fixed license
```